app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -505,8 +597,52 @@ class _CustomChatInputState extends ConsumerState<CustomChatInput> {
         GestureDetector(
           onTap: () {
             inputNotifier.toggleReplyView(false);
+            inputNotifier.toggleEditView(false);
+            inputNotifier.setReplyWidget(null);
+            inputNotifier.setEditWidget(null);
+            inputNotifier.setRepliedToMessage(null);
+            inputNotifier.setEditMessage(null);
+            FocusScope.of(context).unfocus();
+          },
+          child: const Icon(
+            Atlas.xmark_circle,
+            color: Colors.white,
```

Comment:
Avoid hardcoding theme variables.

Reply:
Actually, I have copies this same from the current code base which was already implemented for replyView.

Reply:
Ah, I just see that default Icon Theme have white color so just removed  `color: Colors.white,`.